Scripts/build-derived-assets.sh

```diff
@@ -1,19 +1,56 @@
 #!/bin/sh -eu
 
+#
 #  build-derived-assets.sh
 #  Loop
 #
 #  Copyright © 2019 LoopKit Authors. All rights reserved.
+#
+
+SCRIPT="$(basename "${0}")"
 
+error() {
+  echo "ERROR: ${*}" >&2
+  echo "Usage: ${SCRIPT} <directory>" >&2
+  echo "Parameters:" >&2
+  echo "  <directory>  directory with derived assets" >&2
+  exit 1
+}
 
-echo "Building DerivedAssets.xcassets"
+info() {
+  echo "INFO: ${*}" >&2
+}
+
+if [ ${#} -lt 1 ]; then
+  error "Missing arguments"
+fi
 
-output="${PROJECT_DIR}/Loop/DerivedAssets.xcassets"
+DIRECTORY="${1}"
+shift 1
 
-override="${PROJECT_DIR}/../AdditionalAssets.xcassets/."
```

Comment:
Moving the location of this will break how DIY users are currently using workspace builds. Looks like DIY will need a script to copy files into place like the prepare script in the Tidepool LoopWorkspace.

Reply:
Yes, this will need an update in DIY.